docs/api/product_variants.rst

```diff
@@ -0,0 +1,738 @@
+Product Variants API
+====================
+
+These endpoints will allow you to easily manage product variants. Base URI is `/api/v1/products/{productId}/variants/`.
```

Comment:
But we use everywhere camelCase. Therefore I'm not sure that it should be changed.